extension/src/providers/gotoProvider.ts

```diff
@@ -8,7 +8,7 @@ import { SearchPatterns, SearchHandlers } from './providerShared';
 
 export class AutolispDefinitionProvider implements vscode.DefinitionProvider{
 	async provideDefinition(document: vscode.TextDocument, position: vscode.Position, token: vscode.CancellationToken): Promise<vscode.Location | vscode.Location[]> {
-		const rDoc = ReadonlyDocument.getMemoryDocument(document);
+		const rDoc = AutoLispExt.Documents.getDocument(document);
 		let selected = '';
```

Comment:
For the line 32, I still want to filter out to re-search in the `ActiveDocument`

Reply:
I think we'll ultimately wind up as much and more likely a lot more work filter that out than it currently would failing the simple pre-check.

Reply:
got it!